flocker/control/schema/types.yml

```diff
@@ -45,6 +45,32 @@ definitions:
     maxProperties: 16
     additionalProperties: false
 
+  restart_policy:
+    title: "Container restart policy"
+    type: object
+    description: "Policy for whether or not a container should be restarted after exit."
+    oneOf:
+    - additionalProperties: false
+      properties:
+        maximum_retry_count: {minimum: 1, type: integer}
```

Comment:
Sounds like Jean-Paul wants us to use consistent style, so switch this to indentation based.